aws/cloudformation-templates/base/notebook.yaml

```diff
@@ -206,7 +226,55 @@ Resources:
                 Action: 
                   - lambda:GetFunction
                 Resource: !Sub 'arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:RetailDemoStorePinpointRecommender'
-  
+        -
+          PolicyName: "7-Location"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              -
+                Effect: "Allow"
+                Action:
+                  - geo:*
+                Resource: "*"
```

Comment:
I think for the notebook one since it is a playground, best to leave permissions broad? We are working on scoping down on the Cognito auth and the Location resource creation Lambda though.

Reply:
It's best practice (and actually a requirement for AWS projects like this) to scope down all permissions. Thanks.